dts/bindings/rtc/nxp,irtc.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: IRTC
+
+compatible: "nxp,irtc"
+
+include: rtc.yaml
+
+properties:
+  reg:
+    required: true
+
+  clock-select:
+    type: int
+    required: true
+    description: |
+      The input clock select for IRTC.
+      0 <- 16.384 kHz
+      1 <- 32.768 kHz
+
+  alarms-count:
+    type: int
+    required: true
+    description: Number of alarms supported by IRTC.
+
+  output-clk-en:
+    type: boolean
+    description: |
+      The IRTC can send its clock as an output source
```

Comment:
what is an output source? what does "send" the clock mean?

Reply:
This is the reason why I made the original description more complicated, the output source is the freq that is being used to control the RTC Counter, this can be gated which can disable output for other peripherals using this variable.